spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/config/DataFlowClientProperties.java

```diff
@@ -0,0 +1,76 @@
+package org.springframework.cloud.dataflow.rest.client.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.dataflow.core.DataFlowPropertyKeys;
+
+/**
+ * Configuration properties used in {@link DataFlowClientAutoConfiguration}
+ *
+ * @author Vinicius Carvalho
+ */
+@ConfigurationProperties(prefix = DataFlowPropertyKeys.PREFIX + "client")
+public class DataFlowClientProperties {
+
+	private String target = "http://localhost:9393";
```

Comment:
The property name should be `uri` as that matches better with the existing usage in the shell, e.g. 

```
$ java -jar spring-cloud-dataflow-shell-1.3.0.M3.jar --help
Data Flow Options:
  --dataflow.uri=<uri>                              Address of the Data Flow Server [default: http://localhost:9393].
  --dataflow.username=<USER>                        Username of the Data Flow Server [no default].
  --dataflow.password=<PASSWORD>                    Password of the Data Flow Server [no default].
  --dataflow.credentials-provider-command=<COMMAND> Executes an external command which must return an OAuth Access Token [no default].
  --dataflow.skip-ssl-validation=<true|false>       Accept any SSL certificate (even self-signed) [default: no].$ java -jar spring-cloud-dataflow-shell-1.3.0.M3.jar --help
Data Flow Options:
  --dataflow.uri=<uri>                              Address of the Data Flow Server [default: http://localhost:9393].
  --dataflow.username=<USER>                        Username of the Data Flow Server [no default].
  --dataflow.password=<PASSWORD>                    Password of the Data Flow Server [no default].
  --dataflow.credentials-provider-command=<COMMAND> Executes an external command which must return an OAuth Access Token [no default].
  --dataflow.skip-ssl-validation=<true|false>       Accept any SSL certificate (even self-signed) [default: no].
```